_docs/changelog.md

```diff
@@ -7,7 +7,11 @@ permalink: /docs/changelog/
 {:toc}
 
 Current version of the API is `1`
-Current version of the SDKs are `1.12.2`
+Current version of the SDKs are `1.12.3`
```

Comment:
We're currently at v `1.13.0` [in the API spec](https://github.com/transferzero/api-documentation/pull/53/files), so `1.13.1` might be more appropriate if you wanted to keep this as a patch version? I appreciate our versioning is not always consistent, but the scope of the changes here are significant and probably make it worthy of a `1.14.0` release.

cc @emanuelegorga @sztupy

Reply:
That is correct @archmagos. Also last time we generated the SDKs with version `1.13.0` and we didn't release them (the last released version is `1.12.2`). We could potentially generate the SDKs with the change we need and keep `1.13.0` version in the swagger config files, so release them as `1.13.0`.

Reply:
@archmagos thanks for pointing that out! I will rebase and update the version if we decide to go ahead with this integration. It's currently on hold though, see https://github.com/bitpesa/bitpesa-api-specification/pull/253#discussion_r593110423

Reply:
No problem @mchochowski. I'll also be adding a change soon so will combine with Emanuele's updates and release everything as v `1.13.0`. You should be free to use `1.14.0` etc if the integration goes ahead.

@emanuelegorga just to confirm, this is the addition of `GHS::Cash` which hasn't been released yet? Is there anything blocking a release if so?

Reply:
The addition of the `GHS::Cash` wasn't actually strictly needed as we do already support all its fields with the old version of the SDK so that is why we did not release it. We should just be free to include it with any release and there should not be any problem.

Reply:
Noted, thanks @emanuelegorga